lib/tablex.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
-module.exports = function (buffer){
-  var data = new DataView(buffer);
-  var offset = 8;
+module.exports = function (buffer, bufferx){
+  var tbl_num_rows = (new DataView(buffer)).getUint32(4,true);
```

Comment:
Style nit, camelCase variable name, space after the coma and I belive the parentheses are Un needed due to the new operator having higher precedence the the dot operator.